### DISCUSSION TITLE
矩阵存储·Matrix Storage，Web3时代的存储新选择

### DISCUSSION THREAD

**Discussion by DaYu-IPFS** [filecoin-project/community#124] [News and Events]
“矩阵存储·Matrix Storage"以IPFS&Filecoin网络作为存储技术底层，为有效数据存储提供一个“高安全”、"高可靠"、"低成本"的选择。在产品实现环节，矩阵存储“提供简单易用的PaaS服务和OpenAPI服务。矩阵存储将依托雅典娜云池规模化存储能力，为有数据需求的客户提供支持。
中国消费级基因检测第一的互联网+基因检测企业「23魔方」、中国领先的大健康解决方案企业「咕咚健康」等已成为“矩阵存储·Matrix Storage”的首批深度合作伙伴。

**Comment by MatrixStorage** [comment]
[Matrix storage] update to V1.0.1!
Updates:
More fluency in uploading
1. API adds sharding function while uploading files. You can choose the sharding size while uploading and synthesize a complete file to form a CID after completely uploading. The single uploading file jumped from 512MB to 32GB.
2. Multiple storage control options are available, closely connect with the actual business, ensuring data privacy and improving efficiency.
3. Achieve multiple uploading of the same file.
4. Optimize the total progress bar when uploading files, which make you grasp the overall progress of uploading.
5. Keep the display of all the uploaded files.
 
Better visual experience
1. Optimize the display of API document, more concise and more powerful
2. Optimize all buttons
Entrance for Quick Application ➡ ️
https://storage.anmaicloud.com
------------------------
[矩阵存储]更新到V1.0.1!
更新:
更流畅的上传
1. API在上传文件时增加了分片功能。上传时可以选择分片大小，上传完成后合成一个完整的文件形成CID。单个上传文件从512MB跃升至32GB。
2. 多种存储控制选项，紧密联系实际业务，确保数据私密性，提高效率。
3.实现同一个文件的多次上传。
4. 上传文件时优化总进度条，让您掌握上传的整体进度。
5. 保持显示所有上传的文件。
更好的视觉体验
1. 优化API文档显示，更简洁，功能更强大
2. 优化所有按钮
快速申请入口 ➡ https://storage.anmaicloud.com

**Comment by MatrixStorage** [comment]
🎉「 Matrix storage 」update to V1.1.0! 🎉 
-- -- -- -- -
❤️ Better experience in uploading:
1⃣Uploading files smaller than 1MB are no longer restricted;
2⃣add advanced screening function to get faster retrieval;
3⃣add the automatic retry mechanism of archiving to improve the success rate of archiving;
4⃣add SMS verification to ensure your account security;
5⃣provide English version, and users switch languages smoothly;
👀 More excellent visual  experience: 
1⃣UI optimization of homepage, cooperation page and login page
Entrance for Quick Application（Enterprise Only): https://storage.anmaicloud.com
——————————————————
——————————————————
🎉「 矩阵存储 MatrixStorage 」V1.1.0正式上线！🎉 
—————————
“ 更贴心❤️的上传体验：
1⃣️小于1MB的文件不再受上传限制；
2⃣️增加高级筛选功能，更快检索相关文件；
3⃣️增加归档自动重试机制，提高归档成功率；
4⃣️增加短信验证，为您的账号安全保驾护航；
5⃣️提供英文版本，中/英语言用户使用无缝衔接；
“ 更优秀的视觉👀感受：
1⃣️首页及合作页，登录页的UI优化

企业客户申请体验快速入口️：https://storage.anmaicloud.com


![image](https://user-images.githubusercontent.com/82867405/120579939-e69bc100-c45a-11eb-96ce-9222d740453b.png)

**Comment by MatrixStorage** [comment]
🎉「矩阵存储」更新至V1.2.0🎉

🆕全新功能上线：
1⃣️用户可直接上传NFT文件以及查看

❤️优化登录体验：
1⃣️支持亚马逊S3协议，提升接口可用性；
2⃣️完成注册即可访问系统，完成认证后可进行系统操作；
3⃣️认证完成后可通过注册手机号接受账密信息；
4⃣️支持海外手机号注册；
5⃣️支持手机号变更；
6⃣️界面优化；

企业客户快速入口️：https://storage.anmaicloud.com
-------------------------
🎉Matrix Storage updated to V1.2.0🎉

🆕New functions are in service:
1⃣️Users can directly use EASYNFT to upload and view files

❤️Optimize login experience:
1⃣️Amazon S3 protocol is supported, which increase interface availability;
2⃣️You can access the system after registration, and operate the system after authentication;
3⃣️You can accept the account and password through the registered phone number after authentication;
4⃣️Signing up with overseas phone number is supported;
5⃣️Changing mobile phone number is supported;
6⃣️Interface optimization;
Entrance for Quick Application (Enterprise Only) 👉  https://storage.anmaicloud.com

**Comment by MatrixStorage** [comment]
🎉Matrix Storage updated to V1.2.1🎉
—————————
❤Better experience in using:
1⃣You can enter the adapted WAP pages on mobile phones, and get a better experience without restriction of the device;
2⃣Private storage can be turned into public storage;
3⃣Updated Uploading Agreement and Privacy Policy ensure the security and compliance of documents;

❤More excellent visual  experience:
1⃣UI optimization of console, more clear and concise. 

Entrance for Quick Application (Enterprise Only) 👉 https://storage.anmaicloud.com

🎉「矩阵存储」更新至V1.2.1🎉
—————————
❤️ 更好的使用体验：
1⃣️适配移动端的访问入口，不受设备限制随时体验；
2⃣️可转化的存储类型，支持私有存储转为公有存储； 3⃣️上传、隐私协议更新，保障文件上传安全合规； 

❤️更优的视觉感受：
1⃣️控制台UI优化，更加简洁明了；

企业客户快速入口️：https://storage.anmaicloud.com

![image](https://user-images.githubusercontent.com/82867405/124106090-78b4da80-da96-11eb-80f9-a515c588a5fd.png)

**Comment by MatrixStorage** [comment]
「Matrix storage」update to V1.3.0!

- Fast Uploads
1. Multipart uploads of S3 protocol is supported in OpenAPI
2. Multipart uploads can support up to 5T

- Easy Operation
1. Filter files by format is supported
2. Add storage statistics panel
3. Add file delete function
4. Create new Bucket for more location options


- Simple Experience in Vision
1. UI optimization in console

Entrance for Quick Application (Enterprise Only)   https://storage.anmaicloud.com
---------------------
「矩阵存储」更新到V1.3.0！

- 流畅的上传
1. 支持S3协议的OpenAPI分片上传；
2. OpenAPI分片上传最大上限支持到5T

- 便捷的操作
1. 可通过文件格式进行文件筛选
2. 增加存储数据统计面板
3. 增加了文件删除功能
4. 新建Bucket，增加了区域的选择

- 简洁视觉体验
1. 文件控制台操作界面UI优化

企业快速申请入口  https://storage.anmaicloud.com

![image](https://user-images.githubusercontent.com/82867405/126729470-83e6638f-dcdd-413c-9baa-1d2d71683f35.png)

**Comment by MatrixStorage** [comment]
- MatrixStorage news - 
MatrixStorage was invited to attend the Notary Governance Conference to share the notary Toolset’s features and development direction with official and global notary public. The Notary Toolset records the notary’s DataCap assignment and query, which improves the notary’s efficiency and transparency. The following is the product introduction of the Notary Toolset. I hope you can give me your suggestions on optimization so that the matrix storage product can grow better. Thank you！
https://drive.google.com/file/d/1Q6j25C0Us4YYrZFwN14VjG8BaWHlQ7rM/view?usp=sharing
Join Us：https://storage.anmaicloud.com/index?lang=en
![image](https://user-images.githubusercontent.com/82867405/128283946-a9b2f2b6-784a-4e49-8503-c3d8339be8c2.png)

**Comment by MatrixStorage** [comment]
[MatrixStorage] update to V1.3.1!
-- -- -- -- -
- Better experience in uploading
1. Suspend, resume, or cancel when uploading
2. Support catalog management of files in the Bucket
3. Support file-sharing
4. Support re-archiving
 
- Optimize OpenAPI scheduling
1. Managing APPID and Secret is available
2. Add statistics of OpenAPI consumer usage
3. Add OpenAPI segmented download
 
Entrance for Quick Application (Enterprise Only)   https://storage.anmaicloud.com

---------------------------------------------
「矩阵存储」更新至V1.3.1！
* 优化上传体验
1. 上传文件支持暂停、续传、取消；
2. 支持在Bucket中对文件进行目录管理；
3. 支持文件分享；
4. 支持重新归档；
* 优化OpenAPI调度
1. 可对APPID和Secret进行管理；
2. 增加OpenAPI用户使用情况的数据统计；
3. 增加OpenAPI对文件分块下载；

企业客户快速入口️：https://storage.anmaicloud.com

![MatrixStorage](https://user-images.githubusercontent.com/82867405/129298213-927c64ef-8571-4df7-af51-7340e1ae9b7d.png)

**Comment by MatrixStorage** [comment]
MatrixStorage is updated to the Filecoin Ecosystem as a developer tooling。
![生态地图](https://user-images.githubusercontent.com/82867405/129299240-f7eefa6f-34fd-4c09-8492-85cd3b08162c.png)

**Comment by MatrixStorage** [comment]
Global debut! 
“FIL+ notary report” powered by @MatrixStorage!
The report covers 24 notaries in charge and more than 500 DataCap applications. Relevant DataCap amount is over 1000TiB... This report is detailed with notary distribution, allocation preferences, and DataCap allocation progress, etc. Now, it is available worldwide in Both Chinese and English version for FREE.
You don’t want to miss it! Check it out：https://drive.google.com/file/d/11T4SRoKBLnsHyEM2375xKNaj3U1G_lH2/view?usp=sharing
![image](https://user-images.githubusercontent.com/82867405/129866641-21edf722-2587-481e-a25d-1cf3289b55c3.png)

**Comment by MatrixStorage** [comment]
- Breaking News - 
MatrixStorage “FIL+ Large Data Report” blockbuster release!
- 22 large datasets applications
- 50 PiB data size involved
The report records the Status Quo of LDN since the official launch in April 2021, analyzing applied projects, approval status, approval process, etc. Optimization analysis and future prospects are included as well. For FiL+ Ecology ! Don’t miss this periodic LDN report!

Large Datasets Application Guide
https://drive.google.com/file/d/1LyktUO4jzbLkx2ICWXeWfSrFaqp86v1U/view?usp=sharing
List of Applied Projects
https://docs.google.com/spreadsheets/d/1hlvpH36qQLF1GPyvvraKefuTsUt9yxx7/edit?usp=sharing&ouid=114684011094112462749&rtpof=true&sd=true
Filecoin Plus Large Datasets Periodic Report
https://drive.google.com/file/d/1J__E67B6ym6tgXIv5tOYT8NP1n42HaK_/view?usp=sharing

**Comment by MatrixStorage** [comment]
「矩阵存储」更新到V1.4.0！
- 更好的使用体验
1. 上线去中心化存储的计费模式，便于用户按需选择；
2. 新增“子账号”，主账号可授权子账号进行在线/API等存储操作；
3. 新增“我的钱包”，充值、支出等明细清晰掌握；
4. 新增“账单”功能，使用扣费明细一目了然；
5. 新增“存储类型”和Bucket随机机制，减少选择冗余时间，提升存储效率；
 
企业客户申请体验快速入口️：https://storage.anmaicloud.com
 
 
[MatrixStorage] update to V1.4.0!  
- Better user experience
1. The billing mode of decentralized storage is launched, easy to be chosen as demand.
2. Add sub-accounts. The primary account can authorize sub-accounts to perform online or API storage operations.  
3. Add "My Wallet" to make the details of recharge and expenditure clear;  
4. Add "Bill" function to make the details of use and deduction at a glance;  
5. Add Storage Type and Bucket Random Mechanism to reduce redundancy time and improve storage efficiency.  
   
Entrance for Quick Application (Enterprise Only)   https://storage.anmaicloud.com
![2111631589483_ pic_hd](https://user-images.githubusercontent.com/82867405/133189314-94f07acd-2de6-4bcb-ae41-c6ec4ab97580.jpg)

**Comment by MatrixStorage** [comment]
云响力峰会 | 万物新生·云生多态 🔥
安迈云矩阵存储Matrix Storage官方发布
并与中经天平、咕咚健康、长城超云以及和信创天正式达成战略签约
基于IPFS去中心化存储技术机制
高安全、极隐私、防篡改、难磨灭
以6大核心优势，完备产品架构
成为全球去中心化数据存储市场构建者

New generation cloud computing conference
Matrix Storage OFFICIAL released!🔥
Officially reached strategic cooperation with Zhongjing Tianping, Codoon Health, Super Cloud and Vesystem. 
·Based on IPFS decentralized storage technology
·High security, extremely private, tamper-proof, indelible
·With six core advantages, complete product architecture
·To be the constructor of global decentralized storage market

峰会集锦及【矩阵存储】产品发布详情，请戳↓
Summit Highlights and Product-release Details, please check↓

![产品发布](https://user-images.githubusercontent.com/82867405/134620305-e156c00e-841f-4f3e-87d2-8c4dc955dc7a.jpg)
![战略合作1](https://user-images.githubusercontent.com/82867405/134620316-90662fe7-fef7-4544-a6db-9002ce583225.jpg)
![战略合作2](https://user-images.githubusercontent.com/82867405/134620323-8fc4dc91-492f-41c9-a535-e411898d6b92.jpg)

**Comment by MatrixStorage** [comment]
**「矩阵存储」更新到V1.4.3!**
- 更好的上传体验
1、支持直接上传文件夹，更省心的批量上传方式；
2、上传数据预估费用提示，避免超量计费；
3、传输中心显示上传速率，上传效率一目了然；
- 更便捷的操作
1、官网首页新增新手指南，系统操作熟练掌握；
2、支持汇款等信息填写回显，避免重复操作；
**企业快速申请入口** ： https://storage.anmaicloud.com

**[MatrixStorage] update to V1.4.0!** 
- Better upload experience
1. Support uploading folders directly, worry-less batch upload;
2. Add ‘Estimate cost tips’ while data upload to avoid overcharging;
3. Upload rate is displayed in the mission center, making upload efficiency clear at a glance
- More convenient operation
1. Guidebook is added on the homepage for new users, proficient in system operation
2. Support remittance information auto-complete to avoid repeated operations.
 **Entrance for Quick Application (Enterprise Only)**  ： https://storage.anmaicloud.com


![中文界面](https://user-images.githubusercontent.com/82867405/135052190-a8bccab0-23aa-4c1a-b354-53285e4f1b86.png)
![英文界面](https://user-images.githubusercontent.com/82867405/135052201-42662bbd-dd26-4d88-b840-da84b20bc896.png)

**Comment by MatrixStorage** [comment]
**祝贺！**
矩阵存储正式通过了“中国软件评测中心(CSTC)“技术测评，并获得资格认证，所有重要指标均达到了官方系统需求规格说明书的要求！
作为分布式存储系统，矩阵存储通过专业认证是可圈可点的！这意味着它在产品质量、系统承载能力、安全性等方面已经达到行业领先水平，也标志着它作为成熟产品全面进军分布式存储系统市场。

**Huge congratulations!**
Recently, MatrixStorage has passed the technical test of “China Software Testing Center (CSTC)” and gained the technical certification officially , with all crucial indicators qualified for the demand of the system requirements specification.
As a distributed storage system, receiving the professional certification is a remarkable milestone for MatrixStorage. It means the overall product maturity of the system in product quality, system capacity, system security and etc. has reached the leading level in the industry, and also marks that the system has fully entered the market.

![软件产品测试证书_00](https://user-images.githubusercontent.com/82867405/137079639-a3bc39fc-4935-48b7-b7b3-3a08f54889d5.png)

**Comment by MatrixStorage** [comment]
**【矩阵存储】更新至V1.5.0版！～**
-更实用的体验：
1.增加充值详情，列表仅显示重要信息；
2.可按照存储类型查询数据使用情况；
3.订单和存储容量短信预警；
-更简洁的操作：
1.增加列表筛选，点击关键词即可查询
2.移动版官网上线付费产品信息；
**企业客户申请体验快速入口️**：storage.anmaicloud.com

**[MatrixStorage] update to V1.5.0! ~**
-More practical experience:
1.Add recharge details. Only key information is displayed in the list;
2.Data usage can be queried by storage type;
3.Add SMS warnings for orders and storage capacity;
-More concise operation:
1.Add list filter, click target keywords to query;
2.Payment scheme is added on the mobile terminal.
**Entrance for Quick Application (Enterprise Only)**   ：storage.anmaicloud.com
![WechatIMG6026](https://user-images.githubusercontent.com/82867405/139358608-69be7ad6-62f3-4f01-a73d-bcfcd397c6dc.png)
![WechatIMG6022](https://user-images.githubusercontent.com/82867405/139358618-4e68f7ca-18e1-4328-812f-4d513c3c54b8.png)
![WechatIMG6011](https://user-images.githubusercontent.com/82867405/139358630-89a0691a-3419-4a7d-aca1-8072755577ef.png)
![WechatIMG6010](https://user-images.githubusercontent.com/82867405/139358624-b1ced0f5-7441-4ad1-afca-35b89c19a22d.png)

**Comment by MatrixStorageTech** [comment]
[MatrixStorage] update to V1.5.3
- More convenient storage experience
1. Add batch control of cancel/resume features, reducing complex operations;
2. Add single sign-on (SSO) to avoid system operations affected by multi-point login;
3. Add downstream traffic statistics for clearer consumption;
4. Add Large Dataset Datacap application entrance;
- Cleaner visual design
1. Remove redundant fields from the panel;
Entrance for Quick Application (Enterprise Only)   https://storage.anmaicloud.com

「矩阵存储」V1.5.3版本迭代：
- 更便捷的存储体验
1. 新增批量取消/续传功能，减少复杂操作；
2. 新增账号单点登录，避免多点登录影响系统操作；
3. 新增下行流量统计，流量消耗更清晰；
4. 新增大份额DC申请入口；
- 更简洁的视觉效果
1. 删除控制台冗余字段；
企业快速申请入口：storage.anmaicloud.com
![V1 5 3中文海报](https://user-images.githubusercontent.com/85657288/142095369-449090af-cb7f-42ed-bed2-890820043faf.png)

**Comment by MatrixStorage** [comment]
### **_[MatrixStorage] update to V1.6.0_**
- Easier storage experience
1. Automatic file backup, max to 6 copies at the same time
2. Simpler process of uploading. Farewell complicated options;
- Appropriate visual impression
1. New brand upgrade
2. Official website fitter with the industry standards

-------------------------

### **_【MatrixStorage】更新至V1.6.0_**
- 更轻松的存储体验
1.自动支持文件备份，最多可同时备份6份；
2.简化上传文件流程，告别繁琐选项；
- 更贴近行业的视觉
1.全新品牌升级，
2.更贴近行业的官网页面；

![英文](https://user-images.githubusercontent.com/82867405/145526078-ad356745-bc40-4889-8511-058eb8ba3c91.png)
